.github/workflows/post-merge-ci.yml

```
@@ -205,102 +205,55 @@ jobs:
   # ============================================================================
 
   operator:
```

actionlint (1.7.12)

[error] job "operator" needs job "changed-files" which does not exist in this workflow

(job-needs)